packages/conventional-commit-lint/test/conventional-commit-lint.ts

```diff
@@ -98,7 +100,9 @@ describe('ConventionalCommitLint', () => {
         './pull_request_synchronize'
       ));
       // create a history that has one valid commit, and one invalid commit:
-      const invalidCommits = require(resolve(fixturesPath, './invalid_commit'));
+      const invalidCommits = [
+        ...require(resolve(fixturesPath, './invalid_commit')),
```

Comment:
because Node.js caches `require` I was actually modifying the initial require in each test.